runtime-testsuite/test/org/antlr/v5/test/runtime/CustomDescriptors.java

```diff
@@ -46,16 +45,14 @@ private static RuntimeTestDescriptor getLineSeparatorLfDescriptor() {
 						"[@5,5:4='<EOF>',<-1>,3:1]\n",
 				"",
 				null,
-				"L",
-				"lexer grammar L;\n" +
+				new String [] { "lexer grammar L;\n" +
 						"T: ~'\\n'+;\n" +
-						"SEPARATOR: '\\n';",
-				null, false, false, false,  PredictionMode.LL, true, null, uri);
+						"SEPARATOR: '\\n';" }, null,
```

Comment:
are you sure the right curly brace should also be dropped ?

Reply:
Now `RuntimeTestDescriptor` contains several grammars instead of one (tests infrastructure support combined grammars as well as single (lexer/parser) grammar). So, enclosing within curly braces is required for creating array of strings (grammars).

If I misunderstand you, could you please elaborate?

Reply:
Sorry, GH hallucination...